index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Chocolate Pizza</title>
+</head>
+
+<div class="header_container">
+    <div id= "blogTitle">
+    <img src="https://github.com/alchemycodelab/alchemy-bootcamp-i-january-2019/blob/master/curriculum/week-02/class-09/LAB-10/LAB-10a-chocolate-pizza/lab-assets/logo.png?raw=true">
```

Comment:
Looks like y'all already have a copy of the pictures in your assets folder so you don't need to link it to a url. you could just write a path like this: `assets/logo.png` in the `src` attribute.